frontend/app/services/query-service.js

```diff
@@ -164,10 +164,11 @@ module.exports = function(Query, Sortation, $http, PathHelper, $q, AVAILABLE_WOR
 
     loadAvailableUnusedColumns: function(projectIdentifier) {
       return QueryService.loadAvailableColumns(projectIdentifier)
-        .then(function(available_columns) {
-          availableUnusedColumns = WorkPackagesTableHelper.getColumnDifference(available_columns, QueryService.getSelectedColumns());
-          return availableUnusedColumns;
-        });
+        .then(QueryService.selectUnusedColumns);
```

Comment:
if this is only used within the service, it should not be exposed in the service. could you just add the function to the outer scope, even outside the `QueryService` definition?